src/app/components/ForecastCard/ForecastCard.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ForecastCard from './ForecastCard';
+
+export default {
+  title: 'Component/ForecastCard',
+  component: ForecastCard,
+};
+// Test functions
+
+const getFormattedDate = () => {
+  const today = new Date();
+  let dd: string | number = today.getDate();
+  let mm: string | number = today.getMonth() + 1;
+  const yyyy = today.getFullYear();
```

Comment:
Not a fan of using `dd` `mm` `yyyy` as variable name. Would go simple with `day` `month` `year` ;-)